src/posts/2020-10-15-the-burden-of-choice/index.md

```diff
@@ -0,0 +1,120 @@
+2. Cost of Change as part of a product lifecycle → Usually the one used, just like the image above.
+3. Cost of Change are one of the reason why roadmaps can't be built from a backlog of functionalities you can choose at any given time.
+
+The moment you implement a decision, your future options become more limited, and your speed perhaps slower as well. Any attempt to either go back or move forward will be constrained. You have added to your product new responsibilities. These are now your responsibilities as they are also potential points of failure and anything you do going forward will have to take that into account. It's not entirely avoidable, sure you can try to keep your product as flexible as possible but the effort it will take as some point will be far bigger than the one you can afford (a callback to project management triangle). Hence this is the life of any problem. Let me attempt to parallel it to humankind. In the first years, you learn the basics, from walking to speaking and your personality becomes more and more ingrained. However, at some points, those habits become limiting. Changing becomes harder. Try changing the way you walk now. You can, but you'll frequently fail miserably. 
+
+One of the takeaways from Cost of Change is that Roadmaps can't be set for a long timeframe. Instead, as Giles Colborne compared on [this post](https://www.shopify.co.uk/partners/blog/product-roadmap), roadmaps are like rock climbing. You have a plan of which holds to grab first, as you proceed you choose the next best ones available to you, and quite a few get left behind never touched, while you keep striving towards the Vision and Mission. Often when people think of a Roadmap they expect to see a plan filled with milestones but what tends to be forgotten in those decisions, the choice was to NOT tackle plenty of opportunities. 
+
+<blockquote class="twitter-tweet"><p lang="en" dir="ltr">pm tip<br><br>always make visible the opportunities you didn&#39;t select to pursue ... and the interventions you didn&#39;t select to pursue (and candidates to try next).<br><br>in many ways, this is more helpful context than describing what you actually selected. <a href="https://t.co/LxwDKNFlLq">pic.twitter.com/LxwDKNFlLq</a></p>&mdash; John Cutler (@johncutlefish) <a href="https://twitter.com/johncutlefish/status/1310628556779565056?ref_src=twsrc%5Etfw">September 28, 2020</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script> 
```

Comment:
I think this is supposed to be an embedded tweet

Reply:
I get the idea, but what's the advantage in doing so? If we want to support this then we need to add custom styles for it, otherwise it conflicts with other `blockquote` styles.

Reply:
Embedding a tweet is a rather normal I see in blog posts. If you'd rather not have the possibility of doing so, then I can remove the blockquote and use a markdown version. I copied from here: https://publish.twitter.com/?query=https%3A%2F%2Ftwitter.com%2Fjohncutlefish%2Fstatus%2F1310628556779565056&widget=Tweet

Reply:
I understand but the blog itself has to support this embedding. Like you do for Youtube videos for instance. If our styles don't support it, it won't just work out-of-the-box.

Let it go, I'll work on this after your post. But it will look funny until then.